unfoldingWord/translationCore
# New Windows Installer

#### This pull request addresses:

#1174 It creates a new installer process for Windows, using Inno Installer. 

#### How to test this pull request:

There is a build in the releases for Windows using auto build, check that out and merge.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1303)
